2020-fall/report/ödev1.tex

```diff
@@ -0,0 +1,240 @@
+\documentclass[a4paper]{article} 
+\input{head}
```

Comment:
`head` isimli dosya olmadığı için bu kodu kullanarak PDF elde etmem mümkün değil.